xbmc/platform/android/jni/AudioFormat.cpp

```diff
@@ -24,18 +24,16 @@
 
 using namespace jni;
 
-int CJNIAudioFormat::ENCODING_PCM_16BIT = 0x00000002;
-int CJNIAudioFormat::ENCODING_PCM_FLOAT = 0x00000004;
-int CJNIAudioFormat::ENCODING_AC3       = 0x00000005;
-int CJNIAudioFormat::ENCODING_E_AC3     = 0x00000006;
-int CJNIAudioFormat::ENCODING_DTS       = 0x00000007;
-int CJNIAudioFormat::ENCODING_DTS_HD    = 0x00000008;
+int CJNIAudioFormat::ENCODING_PCM_16BIT                = 0x00000002;
+int CJNIAudioFormat::ENCODING_PCM_FLOAT                = 0x00000004;
+int CJNIAudioFormat::ENCODING_AC3                      = 0x00000005;
```

Comment:
You should initialize to -1, or you never know whether the format is defined or not. If the value is not defined in droid, the value will stay -1.

See https://github.com/koying/xbmc/blob/master/xbmc/android/jni/AudioFormat.cpp#L29 + https://github.com/koying/xbmc/blob/master/xbmc/cores/AudioEngine/Sinks/AESinkAUDIOTRACK.cpp#L224 and following